meadServer/app.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
`const result = await client.query('SELECT batch_name from "BatchData" group by batch_name;');`
This implies you should probably have a batch_table and a foreign key. '1st class' data like that should be handled by the structure of your database in general. In any case, `select distinct batch_name` is probably way faster